common/kotlinx-coroutines-core-common/src/Dispatched.kt

```diff
@@ -11,11 +11,66 @@ import kotlin.jvm.*
 @Suppress("PrivatePropertyName")
 private val UNDEFINED = Symbol("UNDEFINED")
 
+@NativeThreadLocal
+internal object UndispatchedEventLoop {
+    data class State(
```

Comment:
Rename, please, this class are the corresponding state to something other than `State`. It is confusing with `DispatchedContinuation.state`